langchain/embeddings/sentence_transformer.py

```diff
@@ -0,0 +1,63 @@
+"""Wrapper around (chromadb) sentence transformer embedding models."""
```

Comment:
```suggestion
"""Wrapper around sentence transformer embedding models."""
```
This isn't ChromaDB-specific!